src/components/blog/post/body/_anchor.scss

```diff
@@ -0,0 +1,17 @@
+a {
+  &,
+  &:hover,
+  &:link,
+  &:visited {
+    color: #2421ab;
+
+    font-weight: 700;
+    text-decoration: none;
+  }
+
+  &:hover {
+    opacity: 0.5;
+
+    transition: opacity 0.2s;
```

Comment:
Is the designer happy with the default easing? :joy:

Reply:
:man_shrugging: Guess so, this was copied from the Link component (https://github.com/subvisual/subvisual.com/blob/master/src/components/link.module.css) which he never complained about.